vue-frontend/src/components/contributions/WeeklyUpdate.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <div>
+    <div
+      class="tw-container tw-flex tw-flex-col tw-items-center tw-gap-y-2.5 tw-mt-24"
+    >
+      <div>
+        <span
+          class="tw-text-[1.875rem] lg:tw-text-[3.4375rem] tw-leading-[2.4375rem] lg:tw-leading-[5.15625rem] tw-text-black-core/[.87] tw-font-inter-bold"
+        >
+          Our Weekly Tech Updates</span
+        >
+      </div>
+      <div
+        class="tw-mb-8 tw-mt-2.5 lg:tw-mt-6 tw-text-center tw-text-[1rem] lg:tw-text-[1.5rem] tw-leading-[1.5rem] lg:tw-leading-[2.25rem] tw-text-black-core/[0.87] lg:tw-text-black-core/[0.6] tw-font-inter-regular lg:tw-font-inter-medium"
```

Comment:
Is Font color different from lg devices?

Reply:
yes , different opacity in mobile and dektop